pypeit/processimages.py

```diff
@@ -808,6 +804,7 @@ def show(self, attr='stack', idx=None, display='ginga'):
         # Show
         viewer, ch = ginga.show_image(img)
 
+    # TODO: Is this ever used?
```

Comment:
probably by @jhennawi 's stacking code.

Reply:
I don't see it anywhere.  Will comment it for now and see if that breaks anything.